elements/pfe-card/demo/custom-theme.css

```diff
@@ -0,0 +1,233 @@
+
+
+:root {
```

Comment:
Would a smaller subset perhaps be better for testing the implementation?  It will make the demo clearer and easier to debug when inspecting (won't have to sift through all the variables in root if there's a bug).

Reply:
Yep this was a temp file for testing, we could delete it entirely